community/development/cluster-monitoring.md

```diff
@@ -5,7 +5,7 @@
 You need to have the following tools installed on your operating system:
 
 - [`gcloud`](https://cloud.google.com/sdk/docs/install)
-- [`jq`](https://stedolan.github.io/jq/download/) - most distributions have this in repositories
+- [`yq`](https://mikefarah.gitbook.io/yq/#install)
```

Comment:
Maybe it would be good to include the major version, to avoid any confusion. For example:

```suggestion
- [`yq` 4](https://mikefarah.gitbook.io/yq/#install)
```